src/Subscription.hpp

```diff
@@ -27,16 +29,28 @@ namespace impl
  * CLASS DECLARATION
  **************************************************************************************/
 
-template <typename T, typename OnReceiveCb>
+template <typename T>
 class Subscription final : public SubscriptionBase
 {
+
 public:
-  Subscription(Node & node_hdl, CanardPortID const port_id, OnReceiveCb const & on_receive_cb)
+  using OnReceiveCallback_Default= std::function<void(T const &)>;
```

Comment:
I know I suggested this over at #232 but the "problem" with std::function is that it incurs a runtime cost that we'd rather avoid. I only used it as an suggestion because it was easy to convey the point I wanted to make.

At any rate, let's hear @pavel-kirienko 🙏 .